src/server/src/v1/pulse.controller.ts

```diff
@@ -14,4 +14,16 @@ export class PulseController {
     const pulse = await this.activitiesService.getLatestPulses()
     return { message: 'success', data: pulse }
   }
+
+  @Get('time/:startDate/:endDate')
```

Comment:
use url query params instead

Reply:
use nestjs query param with a dto for validation instead of doing it in the service